cmd/generate/codeowners/output.go

```diff
@@ -8,18 +8,30 @@ import (
 	"strings"
 
 	"github.com/open-sauced/pizza-cli/pkg/config"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 )
 
-func generateOutputFile(fileStats FileStats, outputPath string, ownersStyleFile bool, config *config.Spec) error {
+func generateOutputFile(fileStats FileStats, outputPath string, opts *Options, cmd *cobra.Command) error {
 	// Open the file for writing
 	file, err := os.Create(outputPath)
 	if err != nil {
 		return fmt.Errorf("error creating %s file: %w", outputPath, err)
 	}
 	defer file.Close()
+  var flags []string
```

Comment:
The `just format` should also fix this formatting: Golang uses tabs and looks like you have spaces configured.